deltachat-ffi/deltachat.h

```diff
@@ -6284,6 +6284,15 @@ void dc_event_unref(dc_event_t* event);
 
 #define DC_EVENT_WEBXDC_INSTANCE_DELETED          2121
 
+/**
+ * Data received over an ephemeral peer channel.
+ *
+ * @param data1 (int) msg_id
+ * @param data2 (char*) base64-encoded data.
```

Comment:
i think, on the Java-side, it would be even easier:

- we already have a ptr2jbyteArray(ptr, size_in_bytes) in dc_wrapper.c
- in Java, this would result in byte[] - similar to getWebxdcBlob()